lib/rubygems/gemcutter_utilities.rb

```diff
@@ -129,8 +129,8 @@ def update_scope(scope)
 
     say "The existing key doesn't have access of #{scope} on #{pretty_host}. Please sign in to update access."
 
-    email    = ask "   Email: "
-    password = ask_for_password "Password: "
+    email    = ask "Username/email: "
```

Comment:
Should we name the variable `id` or something then?

Reply:
@deivid-rodriguez Quite frankly not sue, because for me 'id' is also not clear enough

But I totally agree that current variant is super bulky and not elegant

Maybe we can find out some more synonims :)

How about `account`?

Reply:
maybe `email_or_username` ... ?

Reply:
Thank you. I'm good with either alternative proposed, also `identifier` if that's more clear? Change it to your liking and I'm happy to approve!